README.md

```diff
@@ -93,10 +93,6 @@ If you are interested, please contact us at zigrazor@gmail.com or contribute to
   - [How to use](#how-to-use)
   - [Example](#example)
   - [Unit-Test Execution](#unit-test-execution)
-    - [OpenSSL installation](#openssl-installation)
-      - [Ubuntu/Debian](#ubuntudebian)
-      - [RedHat/CentOS/RockyLinux/Fedora](#redhatcentosrockylinuxfedora)
-      - [Other System](#other-system)
```

Comment:
Removed all documentation references to `openSSL`. Nobody should be scared anymore